dcsena/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3==1.34.35
+botocore==1.34.35
```

Comment:
We've been using pyproject.toml for packaging lately, it's concise and does all in one place. See https://github.com/Exabyte-io/esse/blob/dev/pyproject.toml, for example.